### PR TITLE
Closes #32

### LINKED ISSUE #32: LogMessage Object and lowLevel methods
**Is your feature request related to a problem? Please describe.**
Currently while coding and debugging i have to use the full blown log method with all its parameters just to test stuff etc. i would love to have a more flexible way for working on my projects. something like some common use methods with fewer parameters and a logmessage object with all parameters and maybe some new methods to filter would be nice.

### PR DESCRIPTION
**Pull request by ThatsNasu**

### CODE REVIEW

File: src/main/java/dev/bitbite/logging/log/ConsoleLog.java [hunk 1]
```diff
@@ -24,22 +25,18 @@ public ConsoleLog() {
...
+			logContent += logMessage.message;
+		}
+		if(logMessage.message != null && logMessage.exception != null) {
+			logContent += " ";
+		}
+		if(logMessage.exception != null) {
+			logContent += logMessage.exception.getMessage()+" in "+logMessage.exception.getStackTrace()[0].getClassName()+":"+logMessage.exception.getStackTrace()[0].getLineNumber();
+		}
+		System.out.println(this.format(logMessage.logLevel, logMessage.category, logContent));
+		if(logMessage.exception != null) Arrays.stream(logMessage.exception.getStackTrace()).skip(1).forEach(e -> { System.out.println(this.format(LogLevels.STACKTRACE, logMessage.category, e.getClassName()+" in line "+e.getLineNumber())); });
```
Comment (netcodedev) [inline src/main/java/dev/bitbite/logging/log/ConsoleLog.java:40]: Please add linebreaks to improve readability

File: src/main/java/dev/bitbite/logging/log/FileLog.java [hunk 2]
```diff
@@ -50,26 +52,19 @@ public FileLog(String path) throws FileNotFoundException {
...
+			logContent += logMessage.message;
+		}
+		if(logMessage.message != null && logMessage.exception != null) {
+			logContent += " ";
+		}
+		if(logMessage.exception != null) {
+			logContent += logMessage.exception.getMessage()+" in "+logMessage.exception.getStackTrace()[0].getClassName()+":"+logMessage.exception.getStackTrace()[0].getLineNumber();
+		}
+		this.writer.println(this.format(logMessage.logLevel, logMessage.category, logContent));
+		if(logMessage.exception != null) Arrays.stream(logMessage.exception.getStackTrace()).skip(1).forEach(e -> { this.writer.println(this.format(LogLevels.STACKTRACE, logMessage.category, e.getClassName()+" in line "+e.getLineNumber())); });
```
Comment (netcodedev) [inline src/main/java/dev/bitbite/logging/log/FileLog.java:67]: same here

**Review by netcodedev — CHANGES_REQUESTED** [review]

File: src/main/java/dev/bitbite/logging/log/ConsoleLog.java (hunk 1 above)
Comment (ThatsNasu) [reply src/main/java/dev/bitbite/logging/log/ConsoleLog.java]: should be done, @see latestCommit

File: src/main/java/dev/bitbite/logging/log/FileLog.java (hunk 2 above)
Comment (ThatsNasu) [reply src/main/java/dev/bitbite/logging/log/FileLog.java]: should be done, @see latestCommit

**Review by netcodedev — APPROVED** [review]

**Pull request merged**